src/OSPSuite.UI/Views/Importer/UnitsEditorView.cs

```diff
@@ -36,6 +36,12 @@ public UnitsEditorView()
          _columnComboBox.EditValueChanged += (s, a) => OnEvent(onColumnComboBoxTextChanged);
       }
 
+      public string SelectedUnit 
```

Comment:
I do not know why we are using something like this instead of usual binding to an object. You are breaking the pattern we are using everywhere. I am going to merge because this PR has been open for too long It would be good to try to address the comments in PR (either to explain why this is done like this or implement the requested/proposed changes) a bit quicker if possible